src/utilities/sql/SqlFile_Impl.cpp

```diff
@@ -983,24 +983,24 @@ namespace openstudio{
         LOG(Warn, "Reporting Total Source Energy with " << *hours << " hrs");
       }
 
-      const std::string& s = "SELECT Value FROM tabulardatawithstrings WHERE \
-                              ReportName='AnnualBuildingUtilityPerformanceSummary' AND \
-                              ReportForString='Entire Facility' AND \
-                              TableName='Site and Source Energy' AND \
-                              RowName='Total Source Energy' AND \
-                              ColumnName='Total Energy' AND \
-                              Units='GJ'";
+      const std::string& s = R"(SELECT Value FROM TabularDataWithStrings
+                                  WHERE ReportName='AnnualBuildingUtilityPerformanceSummary'
+                                  AND ReportForString='Entire Facility'
+                                  AND TableName='Site and Source Energy'
+                                  AND RowName='Total Source Energy'
+                                  AND ColumnName='Total Energy'
+                                  AND Units='GJ')";
       return execAndReturnFirstDouble(s);
     }
 
 
     OptionalDouble SqlFile_Impl::annualTotalCost(const FuelType& fuel) const
     {
       if (fuel == FuelType::Electricity){
-        return execAndReturnFirstDouble("SELECT Value from tabulardatawithstrings where (reportname = 'Economics Results Summary Report') and (ReportForString = 'Entire Facility') and (TableName = 'Annual Cost') and (ColumnName ='Electricity') and (((RowName = 'Cost') and (Units = '~~$~~')) or (RowName = 'Cost (~~$~~)'))");
+        return execAndReturnFirstDouble("SELECT Value from TabularDataWithStrings where (reportname = 'Economics Results Summary Report') and (ReportForString = 'Entire Facility') and (TableName = 'Annual Cost') and (ColumnName ='Electricity') and (((RowName = 'Cost') and (Units = '~~$~~')) or (RowName = 'Cost (~~$~~)'))");
```

Comment:
Starting here it's just casing for `tabulardatawithstrings` -> `TabularDataWithStrings`